thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dashboard/resources/EntityMappingResource.java

```diff
@@ -0,0 +1,116 @@
+package com.linkedin.thirdeye.dashboard.resources;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import javax.ws.rs.DELETE;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.MediaType;
+
+import org.apache.commons.lang3.StringUtils;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.linkedin.thirdeye.client.DAORegistry;
+import com.linkedin.thirdeye.datalayer.bao.EntityToEntityMappingManager;
+import com.linkedin.thirdeye.datalayer.dto.EntityToEntityMappingDTO;
+import com.linkedin.thirdeye.datalayer.pojo.EntityToEntityMappingBean.MappingType;
+
+@Path(value = "/entityMapping")
+@Produces(MediaType.APPLICATION_JSON)
+public class EntityMappingResource {
+
+  private static ObjectMapper OBJECT_MAPPER = new ObjectMapper();
+  private static final DAORegistry DAO_REGISTRY = DAORegistry.getInstance();
+
+  private static final EntityToEntityMappingManager entityMappingDAO = DAO_REGISTRY.getEntityToEntityMappingDAO();
+
+
+  public EntityMappingResource() {
+  }
+
+  @GET
+  @Path("/view")
+  @Produces(MediaType.APPLICATION_JSON)
+  public List<EntityToEntityMappingDTO> viewEntityMappings(
```

Comment:
I would break this into multiple rest API calls instead to overloading one method
- viewAllEntityMappings
- ViewEntityMappingsFrom
- viewEntityMapingsTo